lib/widgets/about_nature40.dart

```diff
@@ -0,0 +1,207 @@
+      style: TextStyle(
+          fontSize: 14.0, fontWeight: FontWeight.w600, color: Colors.black54),
+    );
+
+    Text _nature40ContactAndMore = Text(
+      'Einen detailierteren Überblick über das Projekt, seine Teilprojekte und die verwendete Sensorik finden Sie auf unserer Website, welche unten verlinkt ist. Bei Interesse, Fragen und Anregungen schreiben Sie bitte eine E-Mail an unsere Projektkoordination.',
+      style: TextStyle(
+          fontSize: 14.0, fontWeight: FontWeight.w600, color: Colors.black54),
+    );
+
+    _launchNature40() async {
+      const url = 'https://www.uni-marburg.de/de/fb19/natur40';
+      if (await canLaunch(url)) {
+        await launch(url);
+      } else {
+        throw 'Could not launch $url';
+      }
+    }
+
+    _launchLOEWE() async {
+      const url =
+          'https://wissenschaft.hessen.de/wissenschaft/landesprogramm-loewe';
+      if (await canLaunch(url)) {
+        await launch(url);
+      } else {
+        throw 'Could not launch $url';
+      }
+    }
+
+    return Card(
+      child: Column(
+        children: [
+          SizedBox(
+            height: 5,
+          ),
+          Row(
+            children: [
+              Text(
+                ' Natur 4.0 | Sensing Biodervisity ',
+                style: TextStyle(
+                    fontSize: 20.0,
+                    fontWeight: FontWeight.w600,
+                    color: Colors.black),
+                textAlign: TextAlign.center,
+              ),
+            ],
+          ),
+          Row(
+            children: [
+              Container(
+                padding: EdgeInsets.all(8.0),
+                width: MediaQuery.of(context).size.width * 0.9,
+                child: Column(
+                  children: <Widget>[_nature40],
+                ),
+              )
+            ],
+          ),
+          Row(
+            children: [
+              Text(
+                ' Natur 4.0 - kurz und bündig ',
+                style: TextStyle(
+                    fontSize: 20.0,
+                    fontWeight: FontWeight.w600,
+                    color: Colors.black),
+                textAlign: TextAlign.center,
```

Comment:
You can simply wrap the Textfile with a Center() widget. There is no need for a Row with a list with one element as its children. You could also think about creating function for the creation of headers (big titles) and sub headers(with smaller titles) to get rid of redundancy. Otherwise the Text widget is repeated several times with the same TextStyle